lib/wallet/wallet.js

```diff
@@ -1862,40 +1930,40 @@ class Wallet extends EventEmitter {
     const mtx = new MTX();
 
     for (const {prevout, own} of bids) {
-      if (!own)
-        continue;
+        if (!own)
```

Comment:
what happened to the whitespace in this function? :-)

Reply:
Well, some leftovers from switching from altering makeReveal to copying it elsewhere. Will change it back !